### ISSUE TITLE
findtransactions ReadTimeout

### ISSUE THREAD

**Issue by theshock9** [iotaledger/chronicle.rs#69]
## Bug description

Making a request with "findTransactions" for an Address

## Version

Chronicle Permanode Alpha v0.1.0

## Hardware specification

- OS and version: 18.04.1-Ubuntu
- RAM: 12GB
- Number of cores: 4

## Steps To reproduce the bug

i make a call with
```
command = {
  "command": "findTransactions",
  "addresses": [
  "..."
  ]
};
```

## Expected behaviour

Get List of Transactions

## Actual behaviour

Got an Error

## Errors

`[chronicle_api::api::findtransactions][ERROR] CqlError { code: ReadTimeout, message: "Operation timed out for tangle.edge - received only 0 responses from 1 CL=ONE.", additional: Some(ReadTimeout(ReadTimeout { cl: One, received: 0, blockfor: 1, data_present: 0 })) }`

**Comment by LouayKamel** [comment]
ReasTimeout is Scylla error when the system is really busy ,currently we don't retry when such an error occurs.

**Comment by LouayKamel** [comment]
@theshock9 I detected the bug, unfortunately all the findtransactions sent to one scylla shard, that's why you got ReadTimeout.

**Comment by theshock9** [comment]
Thx for that Info.
I increased the TimeOut Settings in scylla.yaml and now its all fine.
It was over 1000TX in this Address °-°

**Comment by LouayKamel** [comment]
Fixed in [next-0.1](/iotaledger/chronicle.rs/tree/next-0.1) branch;
